test/action_test.dart

```diff
@@ -26,62 +26,48 @@ void main() {
 
     setUp(() {
       action = Action<String>();
+      addTearDown(() async {
+        await action.dispose();
+      });
     });
 
     test('should only be equivalent to itself', () {
       Action _action = Action();
-      Action _action2 = Action();
+      Action _ActionV2 = Action();
```

Comment:
It looks like this might have been a find/replace issue?

```suggestion
      Action _action2 = Action();
```

Also, I know this isn't your fault, but it feels like the underscores on these local variables are unnecessary. Might be nice to remove them if you need to make a change to this test.

Reply:
oops! yes find and replace